src/canisters/governance/request.converters.ts

```diff
@@ -784,6 +784,24 @@ export const toRegisterVoteRequest = ({
   neuron_id_or_subaccount: [],
 });
 
+export const toMakeProposalRawRequest = (
+  request: MakeProposalRequest
+): RawManageNeuron => {
+  const rawCommand: RawCommand = {
```

Comment:
not related to your PR, more a reminder to my self: we should have a look if we can refactor all these commands, pretty sure there is space for saving a bit of code

Reply:
yep